benchmarks/query_benchmarks/query_get_or_create/benchmark.py

```diff
@@ -0,0 +1,18 @@
+from ...utils import bench_setup
+from .models import Book
+
+
+class QueryGetOrCreate:
+    def setup(self):
+        bench_setup(migrate=True)
+        self.next_id = Book.objects.count() + 1
+
+    def teardown(self):
+        Book.objects.all().delete()
+
+    def time_query_get_or_create(self):
+        # This will do a create ...
+        Book.objects.get_or_create(id=self.next_id, defaults={"title": "hi"})
+
+        # ... and this a get.
+        Book.objects.get_or_create(id=self.next_id, defaults={"title": "hi"})
```

Comment:
This one is rather quick too. Could we repeat this 2x / 3x? That should get us into the `ns` range. 

https://github.com/smithdc1/django-asv/runs/6720503551?check_suite_focus=true#step:5:63